backend/api/projects.py

```diff
@@ -0,0 +1,187 @@
+"""Main Module of knex
+Defines API points and starts the application
+"""
+
+import os
+import sys
+import time
+import json
+import json5
+
+from flask import request, jsonify, make_response, g, Blueprint
+from flask_security import login_required, roles_required
+from werkzeug.utils import secure_filename
+from bson.json_util import dumps
+
+from api.helper import uploader
+from api.helper.apiexception import ApiException
+
+
+projects = Blueprint('api_projects', __name__)
+
+# @login_required
+@projects.route('/api/projects', methods=['POST'])
+def add_project():
+    """Receive manifest as a jsonstring and return new ID
+    """
+    successful_ids = []
+    unsuccessful_files = []
+    uploaded_files = request.files.getlist("file[]")
```

Comment:
ehh, not on topic but before I forget it this should probably be 'file[]' to fit the rest of the code